src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -27,6 +27,15 @@
 			</div>
 		</div>
 
+		<div class="row">
+			<quick-filters
+				class="tw-ml-2"
+				v-if="filtersLoaded && enableQuickFilters"
```

Comment:
Did  the `row` add any vertical space when enableQuickFilters is off? If so, please move this if statement up

Reply:
We gonna fix this on next PRs having all the dropdowns working. Thanks!